app/components/elements/forms/repeatable_nested_component.rb

```diff
@@ -16,6 +16,7 @@ def initialize(form:, model_class:, field_name:, form_component:, hidden_label:
         @reorderable = reorderable
         @single_field = single_field
         @fieldset_classes = fieldset_classes
+        @skip_tooltip = skip_tooltip
```

Comment:
Can you add a comment about what skip tooltip means?

Reply:
Done.